packages/calcite-components/src/components/list-item-group/list-item-group.scss

```diff
@@ -3,44 +3,28 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-list-item-group-background-color: Specifies the background color of the component.
- * @prop --calcite-list-item-group-border-color: Specifies the border color of the component.
- * @prop --calcite-list-item-heading-text-color: Specifies the text color of the component's heading.
+ * @prop --calcite-list-item-group-background-color: Specifies the background color of the component. Will fallback to --calcite-list-background-color.
```

Comment:
Do we need to list the fallback CSS Props in the doc? Also, it looks like the documented fallback props are not used.

Reply:
We do not. Thanks for the catch!